database-commons/src/main/java/io/cdap/plugin/db/sink/ETLDBOutputFormat.java

```diff
@@ -67,14 +72,27 @@ public RecordWriter<K, V> getRecordWriter(TaskAttemptContext context) throws IOE
     String tableName = dbConf.getOutputTableName();
     String[] fieldNames = dbConf.getOutputFieldNames();
     final int batchSize = conf.getInt(COMMIT_BATCH_SIZE, DEFAULT_COMMIT_BATCH_SIZE);
+    final String operationName = conf.get(OPERATION_NAME);
+    String relationTableKey = null;
```

Comment:
Why default access modifier?

Reply:
access modifier are not allowed here as it is inside a public method which is getRecordWriter.